flytekit/annotated/context_manager.py

```diff
@@ -86,16 +86,42 @@ class InstanceVar(object):
     o: Any
 
 
+class EntrypointSettings(object):
```

Comment:
Data class?

Reply:
done, had it originally and then changed when using the env var and reverted back